source/user-manual/reference/ossec-conf/labels.rst

```diff
@@ -12,7 +12,7 @@ labels
 	    <labels>
 	    </labels>
 
-The labels section of ``ossec.conf`` allows additional user-defined information about agents to be included in alerts. When email notifications are enabled, this additional data is also contained in the email alerts without any further configuration.
+The labels section of ``agent.conf`` allows additional user-defined information about agents to be included in alerts. When email notifications are enabled, this additional data is also contained in the email alerts without any further configuration.
```

Comment:
Labels section is also available in `manager.conf`

Reply:
I guess this is valid:
_The labels section of ``agent.conf`` or ``manager.conf``, allows additional user-defined information about agents to be included in alerts. When email notifications are enabled, this additional data is also contained in the email alerts without any further configuration._